### PR TITLE
feat : added alerts

### PR DESCRIPTION
**Pull request by sach10-create**

### CODE REVIEW

File: components/alerts/src/css/simple-alert.css [hunk 1]
```diff
@@ -0,0 +1,227 @@
+@import url(../../../../css/component.css);
+.warning-button {
```
Comment (humzahasan) [inline components/alerts/src/css/simple-alert.css:2]: Maybe you can refactor this to have a common .button class with all the common property under one class.

File: components/alerts/src/css/simple-alert.css [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+  }
+  80% {
+    transform: translateX(0%);
+  }
+  100% {
+    transform: translateX(-10px);
+  }
+}
+
+/* Success Alert */
```
Comment (humzahasan) [inline components/alerts/src/css/simple-alert.css:109]: Same suggestion as above. Always try and follow DRY principle.

File: components/alerts/src/css/simple-alert.css [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+
+/* Dismiss Button */
+.warning-alert .fa-times-circle {
+  position: absolute;
+  right: 20px;
+  top: 50%;
+  transform: translateY(-50%);
+  color: #ce8550;
+  cursor: pointer;
+}
```
Comment (bhushannaikaroskar) [inline components/alerts/src/css/simple-alert.css:200]: Try to use variables for colors. Rest all good to go.

File: css/simple-alert.css [hunk 4]
```diff
@@ -0,0 +1,227 @@
+@import url(../../../../css/component.css);
+.warning-button {
```
Comment (humzahasan) [inline css/simple-alert.css:2]: Common property can be clubbed into a single class

**Review by bhushannaikaroskar — CHANGES_REQUESTED** [review]
Can I know why you used 2 same CSS files simple-alert.css in two different folder you could have reused.

**Review by humzahasan — COMMENTED** [review]
Code structure and the UI looks amazing. Few refactoring is needed for CSS.

File: components/alerts/src/css/simple-alert.css (hunk 1 above)
Comment (sach10-create) [reply components/alerts/src/css/simple-alert.css:2]: I am trying to create different buttons as of now and will generalize once I finalize the concept of ecommerce website.
Thanks for your valuable input Humza.

File: components/alerts/src/css/simple-alert.css (hunk 3 above)
Comment (sach10-create) [reply components/alerts/src/css/simple-alert.css:200]: There are like 3 colors for each alert, so there are like 12 colors that I am not using anywhere other than the alert.
Also I will be deleting the other file after entire completion of the CL, I am keeping it so I can have the folder structure and and all the CSS in one folder

Comment (bhushannaikaroskar) [reply components/alerts/src/css/simple-alert.css:200]: Ok cool I am approving now. Make sure you structure it properly after getting done.

**Review by bhushannaikaroskar — APPROVED** [review]

**Review by humzahasan — APPROVED** [review]
Good to merge!

**Review by zoef-yt — APPROVED** [review]
Good to merge.

**Pull request merged**